### DISCUSSION TITLE
relational create

### DISCUSSION THREAD

**Discussion by ewilliam** [prisma/prisma#11393] [Q&A] [answered]
gist: https://gist.github.com/ewilliam/a2a3fb9b917fcb3a1b301506a3deb016

id think relation create would work? but looks like i have to create a person record on its own and connect instead?

**Comment by austincrim** [comment, marked as answer]
Hey @ewilliam 👋 ,

I recreated your code locally and it works for me. I did have to change the `hubspot_id` to be a something other than `null` because of your schema.

Could you create a reproduction repository that I can clone and try again?

Thanks!

**Reply by ewilliam** [reply]
jeez!! fixing `hubspot_id` worked for me too - i kept focusing on user <-> person because of the error message. didn't think to rule out the invalid null first!

appreciate @austincrim

**Reply by austincrim** [reply]
Good to hear! I will take a look into why the error message was pointing in the wrong direction.

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
